core_impl/src/main/java/com/google/instrumentation/trace/TraceComponentImplBase.java

```diff
@@ -17,12 +17,16 @@
 
 /** Base implementation of the {@link TraceComponent}. */
 public class TraceComponentImplBase extends TraceComponent {
-  private static final Tracer tracer = Tracer.getNoopTracer();
-  private final BinaryPropagationHandlerImpl binaryPropagationHandler =
+  private static final int TRACE_EXPORTER_BUFFER_SIZE = 32;
```

Comment:
Should these constants be part of TraceParams (or maybe special exporter params?)